src/js/messaging/components/MessageDetails.jsx

```diff
@@ -76,35 +76,34 @@ class MessageDetails extends React.Component {
       </div>
     );
 
-    const inputId = `message-details-${messageId}`;
-    const compactInputId = `compact-${inputId}`;
-
     const compactSentDate = sentDate && (
       <span>{formattedDate(sentDate, { fromNow: true })}</span>
     );
 
     return (
       <div
-          ref="detailsControl"
           className="messaging-message-details-control"
           tabIndex="-1"
-          onBlur={this.hideDetails}
+          onBlur={() => {
+            if (this.state.expandedFull) {
```

Comment:
if the purpose of this is to always hide the details on blur, would it be worth doing something like `this. toggleFullDetails(false)`?, i.e. we can override the toggle action by passing a param (true or false).